ios/NedaaWidget/PrayerCountdownLockScreenView.swift

```diff
@@ -0,0 +1,165 @@
+import WidgetKit
+import SwiftUI
+import Intents
+
+
+struct CountdownLockScreenViewProvider: IntentTimelineProvider {
```

Comment:
A lot of the code here is exactly the same as the other widget provider, maybe we can factor this into a helper or util file

Reply:
If I remember correctly we can't use the same provider for two different widgets